WolvenKit.App/ViewModels/Documents/RedDocumentViewModel.cs

```diff
@@ -279,10 +279,11 @@ protected void AddTabForRedType(RedBaseClass cls)
             var slot = atlas.Slots[0];
             if (slot != null)
             {
-                var file = GetFileFromDepotPath(slot.Texture.DepotPath);
-                if (file != null)
+                if (GetFileFromDepotPath(slot.Texture.DepotPath)?.RootChunk is CBitmapTexture tex)
                 {
-                    TabItemViewModels.Add(_documentTabViewmodelFactory.RDTInkTextureAtlasViewModel(atlas, (CBitmapTexture)file.RootChunk, this));
+                    var tab = _documentTabViewmodelFactory.RDTInkTextureAtlasViewModel(atlas, tex, this);
+                    tab.ChangeEvent += OnPartNameChanged;
```

Comment:
do we need to unsubscribe from this when disposing?

Reply:
Probably, but I have no idea how to do that

Reply:
I think we need to unsubscribe to the event when the tab get's closed, otherwise we leak memory

Reply:
I'd have to store the tab somewhere, right?